src/core.jl

```diff
@@ -17,6 +17,10 @@ if !debug_memory
         actions::Vector
         alive::Bool
         preservers::Dict
+        name::String
+        function Signal(v, parents, actions, alive, pres, name)
+            n=new(v,parents,actions,alive,pres,name)
```

Comment:
No use, maybe it's vestigial from an earlier version where I had autoname in there? can't remember, either way, let's remove it